python-in-edu/resources/models.py

```diff
@@ -4,24 +4,52 @@
 from django.core.mail import send_mail
 from django.urls import reverse
 
-from multiselectfield import MultiSelectField
-
 from mysite.settings import DEFAULT_FROM_EMAIL
-from . import choices
 
 
+def get_initial_status():
```

Comment:
This is raising issue while migration. I think the issue is because of the fact that `ResourceStatus` won't exist until this point?
I have attached the screenshot herewith. 
![resource](https://user-images.githubusercontent.com/4557587/120923055-bf4c2a80-c6cc-11eb-9ece-f5dc61d76e2b.jpg)